### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by jainpatel25**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1943"></a>

### CODE REVIEW

File: public/data/jainpatel.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Jain Patel",
+    "type": "personal",
+    "bio": "learning code. I am open to learn new skills",
+    "avatar": "",
```
Comment (kumarsonsoff3) [inline public/data/jainpatel.json:5]: Please use a valid Avatar link! 🛠
```suggestion
    "avatar": "https://github.com/jainpatel25.png",
```

File: public/data/jainpatel.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/jainpatel25",
+        "icon": "github"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "focus on web dev and opensource",
+        "date": "October 2022",
+        "icon": "",
```
Comment (kumarsonsoff3) [inline public/data/jainpatel.json:17]: Use a valid icon here from the list of icons available in our docs [icon.md](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md)

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please make the following changes as suggested below, let us know if you need any help!

**Review by imran1509 — COMMENTED** [review]
Please use a valid icon from the list of icons suggested by  @kumarsonsoff3 
other than that everything looks good to me

**Comment by neerajrathore** [comment]
changes looks fine

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, 
Thank you for raising the PR and for your contribution. 

Sad to say we can't merge this PR, as we have done huge changes to this repo ([**Migrated to Next.js**](https://github.com/EddieHubCommunity/LinkFree/releases/tag/v1.1.0)), and the path to profile addition has been changed. So please raise a new PR and we will try to merge it ASAP. 

You can find the updated instruction with the new path in the [README](https://github.com/EddieHubCommunity/LinkFree#-contributing). If you have any doubts feel free to reach us out on the [EddieHub Discord](http://discord.eddiehub.org/)

NOTE: Fetch/sync/pull new changes then go for the PR.

**Thank you**

**Pull request closed without merging**